smoke-tests/apps/Logback/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/LogbackTest.java

```diff
@@ -102,7 +102,7 @@ void test() throws Exception {
     }
 
     if (!isWildflyServer()) {
-      assertThat(md3.getProperties()).containsEntry("Marker", "aMarker");
+      assertThat(md3.getProperties()).containsEntry("Marker", "[aMarker]");
```

Comment:
let's change this to comma-separated values (without the brackets), so that there's no breaking change

Reply:
ok